### PR TITLE
Escape quotes when using HTML inside CDATA strings

### PR DESCRIPTION
**Pull request by luk1337**
# AniTrend Pull Request

Thank you for contributing! Please take a moment to review our [**contributing guidelines**](https://github.com/AniTrend/anitrend-app/blob/master/CONTRIBUTING.md)
to make the process easy and effective for everyone involved.

**Please open an issue** before embarking on any significant pull request, especially those that
add a new library or change existing tests, otherwise you risk spending a lot of time working
on something that might not end up being merged into the project.

Before opening a pull request, please ensure:

- [x] You have followed our [**contributing guidelines**](https://github.com/AniTrend/anitrend-app/blob/master/CONTRIBUTING.md)
- [x] double-check your branch is based on `develop` and targets `develop` 
- [ ] Pull request has tests
- [x] Code is well-commented, linted and follows project conventions
- [ ] Documentation is updated (if necessary)
- [x] Description explains the issue/use-case resolved
- [x] I did not commit files that are excluded in the .gitignore file (Happens if you stage files with Android Studio)

## Description
Turns out that unescaped quotes get totally stripped out
during compilation which in the end led to a broken link
in "Update AniTrend?" dialog.

## Motivation and Context
I noticed that there was a broken link in Update AniTrend so I tried to find out why and after printing HTML content passed to RichMarkdownTextView I noticed that all quotes are missing.

## How Has This Been Tested?
I opened update window before and after this change.

## Screenshots (if appropriate):
ADB Screenshot from 2021-04-13 10-14-01
ADB Screenshot from 2021-04-13 10-14-27

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

**IMPORTANT**: By submitting a patch, you agree to allow the project
owners to license your work under the terms of the [MIT License](https://github.com/AniTrend/anitrend-app/blob/master/LICENSE.md).

### CODE REVIEW

**Comment by luk1337** [comment]
Should I also fix translations?

**Review by wax911 — APPROVED** [review]

**Comment by wax911** [comment]
Thanks again 💯 and sure you can apply the same to the translations, I'll leave this PR open

**Comment by luk1337** [comment]
Sure, force pushed with updated translations.

**Review by wax911 — APPROVED** [review]

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -160,7 +160,7 @@
         <![CDATA[
         <p>
         <b>AniTrend</b> a échoué à se connecter au serveur plus d\'une fois ! Merci de vérifier que votre connection internet fonctionne
-        et de vérifier que le <a href="https://anilist.co"><b>site Internet d\'Anilist</b></a> est disponible avant d\'appuyer sur "Oui" !
+        et de vérifier que le <a href=\"https://anilist.co"><b>site Internet d\'Anilist</b></a> est disponible avant d\'appuyer sur "Oui\" !
```
Comment (luk1337) [inline app/src/main/res/values-fr/strings.xml:163]: sed failed here. fixing.

**Pull request merged**